plugins/pulp_ostree/plugins/lib.py

```diff
@@ -326,13 +341,35 @@ def import_key(self, path, key_ids):
         :param key_ids: A list of key IDs.
         :type key_ids: list
         """
-        self.repository.open()
+        self.open()
         lib = Lib()
         fp = lib.Gio.File.new_for_path(path)
         in_str = fp.read()
-        imported = self.repository.impl.remote_gpg_import(self.id, in_str, key_ids)
+        imported = self.impl.remote_gpg_import(self.id, in_str, key_ids)
         return imported
 
+    @wrapped
+    def list_refs(self):
+        """
+        Get (remote) repository references.
+
+        :return: list of: Ref
+        :rtype: list
+        :raises LibError:
+        """
+        _list = []
+        lib = Lib()
+        flags = lib.OSTree.RepoPullFlags.COMMIT_ONLY
+        self.open()
+        _, summary = self.impl.remote_list_refs(self.id)
```

Comment:
Just to confirm my understanding of how the GObject introspection works, `_` is the return value of the C function call, and summary is `GHashTable **out_all_refs` (as a python dictionary), correct?